File: solutions/062.unique-paths/unique-paths.java
class Solution {

    public int uniquePaths(int m, int n) {
        int[][] result = new int[m][n];
        for(int i= m-1; i >= 0; i--){
            for(int j = n-1; j >= 0; j--){
                if(i==m-1 || j == n-1) {
                  result[i][j]=1;  
                }else{
                    result[i][j] = result[i+1][j] + result[i][j+1];
                }
            }
        }
        return result[0][0];
    }
}

File: solutions/062.unique-paths/unique-paths.js
/**
 * @param {number} m
 * @param {number} n
 * @return {number}
 */
var uniquePaths = function(m, n) {
    const res = new Array(n);
    for(let i = 0; i < n; i++){
        res[i] = new Array(m);
        res[i].fill(1);
    }
    for(let i = 1; i < n; i++){
        for(let j = 1; j < m; j++){
            res[i][j] = res[i - 1][j] + res[i][j -1];
        }
    }
    
    return res[n - 1][m - 1];
    
};